src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewCell.swift

```diff
@@ -87,8 +87,8 @@ extension DynamicCell {
 }
 
 extension DynamicCell {
-	static func dynamicType(text: String, cellStyle: TextCellStyle = .label, size: CGFloat = 17, weight: UIFont.Weight = .regular, style: UIFont.TextStyle = .body, color: UIColor? = nil, accessibilityIdentifier: String? = nil, accessibilityTraits: UIAccessibilityTraits = .staticText, configure: CellConfigurator? = nil) -> Self {
-		.identifier(cellStyle.reuseIdentifier, action: .none, accessoryAction: .none) { viewController, cell, indexPath in
+	static func dynamicType(text: String, cellStyle: TextCellStyle = .label, size: CGFloat = 17, weight: UIFont.Weight = .regular, style: UIFont.TextStyle = .body, color: UIColor? = nil, accessibilityIdentifier: String? = nil, accessibilityTraits: UIAccessibilityTraits = .staticText, action: DynamicAction = .none, configure: CellConfigurator? = nil) -> Self {
+		.identifier(cellStyle.reuseIdentifier, action: action, accessoryAction: .none) { viewController, cell, indexPath in
```

Comment:
shouldn't be the assassoryAction be adjusted accordingly or is there a different purpose behind it?

Reply:
It could be added as a parameter as well, but since it's not yet used anywhere, I just left it out. Quickly added it now for consistency 👍

Reply:
:-)